local-cli/generator-wpf/templates/src/App.xaml.cs

```diff
@@ -36,6 +37,8 @@ protected override void OnStartup(StartupEventArgs e)
         /// <param name="arguments"></param>
         private void OnCreate(string[] arguments)
         {
+            LauncherModule.SetActivatedUrl(String.Join(" ", arguments));
```

Comment:
Why is this needed in the template?  The arguments are already passed into the ReactPage.OnCreate and should be available on the other side of the bridge... @rozele can you confirm the arguments are available on the javascript side of the bridge?

The choice to set the activated Url is an application decision that I don't believe is relevant to most users... In this case, it seems that the URL being set is the list of arguments... I don't believe this belongs in the template unless I'm missing something...